src/Apps/Artwork/Components/ArtworkBrowser/ArtworkBrowser.tsx

```diff
@@ -126,6 +91,78 @@ export class LargeImageCarousel extends React.Component<ImageBrowserProps> {
   }
 }
 
+interface ArtworkBrowserState {
+  isLocked: boolean
+}
+
+export class SmallArtworkBrowser extends React.Component<
+  ArtworkBrowserProps,
+  ArtworkBrowserState
+> {
+  state = {
+    isLocked: false,
+  }
+
+  get settings(): Settings {
+    return {
+      arrows: false,
+      customPaging: () => <PageIndicator />,
+      dots: true,
+      infinite: false,
+      // TODO: Future optimization should it be needed
+      // lazyLoad: "ondemand",
+      speed: 500,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+    }
+  }
+
+  lock = () => {
+    this.setState({
+      isLocked: true,
+    })
+  }
+
+  unlock = () => {
+    setTimeout(() => {
+      this.setState({
+        isLocked: false,
+      })
+    }, 10)
+  }
```

Comment:
Will need to qa this a bit more on a real device but will check once we send out to staging.